guides/building/testing.html

```diff
@@ -1,7 +1,9 @@
 <html lang="en">
 
 <head>
-  <title>Kitura "Hello World" example</title>
+  <title>Learn - Test Your Kitura App</title>
```

Comment:
Did you mean to include these changes as well?

Reply:
No I didn't they're not a part of this PR, I'll remove them now